### ISSUE TITLE
[prefer-to-be] does not consider template literals as a string

### ISSUE THREAD

**Issue by sapphi-red** [jest-community/eslint-plugin-jest#998]
Template literals are same as normal strings unless they are used as a tagged template. But the prefer-to-be rule does not treat it as same as normal strings. 

```js
const a = 'foo'
expect(a).toEquals(`bar`)
```
The code above should be warned to use `toBe`.
